tools/svsm/Launch_SVSM.sh

```diff
@@ -0,0 +1,596 @@
+#!/bin/bash
+#  SPDX-License-Identifier: MIT
+#
+#  Copyright (C) 2023 Advanced Micro Devices, Inc.  
+
+
+set -eE
+set -o pipefail
+[ "$DEBUG" == 'true' ] && set -x
+
+#trap cleanup EXIT
```

Comment:
Do we want to have this commented out? It catches machine interrupts to make sure things are cleaned up before exiting the program.

Reply:
I agree with the comment and have added a cleanup function to Launch_SVSM.sh